pkg/attestation/crafter/api/attestation/v1/crafting_state.go

```diff
@@ -206,7 +206,7 @@ func (m *Attestation_Material) CraftingStateToIntotoDescriptor(name string) (*in
 
 	// string materials don't have an artifact nor container, so a name is not available.
```

Comment:
This probably is not true anymore. If the ID is outside the artifact, string materials would have an ID as well, right?

Reply:
not sure, since it's taking about name, ... let me double check